tests/custom_messages_worker.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 The Emscripten Authors.  All rights reserved.
+ * Emscripten is available under two separate licenses, the MIT license and the
+ * University of Illinois/NCSA Open Source License.  Both these licenses can be
+ * found in the LICENSE file.
+ */
+
+#include <emscripten.h>
+#include <emscripten/em_asm.h>
+
+#include <pthread.h>
+
+EM_JS(void, run_test, (), {
+   function sendMessageToMainThread(cmd, payload) {
+     self.postMessage({
```

Comment:
What is `self` here?

Reply:
The main worker (I believe?), but it appears to be useless, so let's remove it

Reply:
removed